src/test/java/com/synopsys/integration/jenkins/wrapper/JenkinsProxyHelperTest.java

```diff
@@ -150,6 +151,42 @@ public void testFromJenkinsValidProxyInfoJenkins(String url, List<Pattern> ignor
         validProxyInfoValidation(proxyInfo, proxyHost, proxyPort, proxyUsername, proxyPassword, testNtlmDomain, StringUtils.EMPTY);
     }
 
+    @ParameterizedTest
+    @MethodSource({ "validProxyInfo"})
+    public void testFromJenkinsValidProxyInfoNullUserName(String url, List<Pattern> ignoredProxyHosts, String testNtlmDomain) {
+        ProxyConfiguration proxyConfigurationMock = Mockito.mock(ProxyConfiguration.class);
+        Jenkins jenkinsMock = Mockito.mock(Jenkins.class);
+        JenkinsWrapper jenkinsWrapper = new JenkinsWrapper(jenkinsMock);
+
+        setJenkinsFields(jenkinsMock, proxyConfigurationMock);
+
+        Mockito.when(proxyConfigurationMock.getUserName()).thenReturn(null);
```

Comment:
All other valid positive tests used a valid/populated property to return for getUserName. This was flagged by Jacoco branch coverage, so this tests returns a null for getUserName and then validates the same is returned by validProxyInfoValidation().

Should also be noted that a null userName also requires that the PW is empty, so the return for getPassword() and the validation for PW is adjusted accordingly.

Reply:
In review with @rottebds, it was pointed out that better use of input method params could be implemented which would add the coverage that this test was specifically doing. This test has been removed.